src/pages/landing/forex-trading/components/_hero.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex, Container, Show } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { Background } from 'components/elements/background-image'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+
+const BackgroundWrapper = styled(Background)`
+    background-size: cover;
+    background-position: bottom right;
+`
+
+const Wrapper = styled(Container)`
+    @media ${device.tabletS} {
+        height: 70rem;
+        margin-left: 0;
+        padding: 2rem 16px 0;
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+`
+
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 75rem;
+    z-index: 1;
+
+    @media ${device.tabletL} {
+        max-width: 55rem;
+        top: 280px;
+        padding: 0 16px;
+    }
+    @media ${device.mobileL} {
+        max-width: 390px;
+        padding: 0;
+    }
+`
+
+const StyledHeader = styled(Header)`
+    display: flex;
+
+    @media ${device.tabletL} {
+        margin-top: 1.3rem;
+    }
+    @media ${device.mobileS} {
+        margin-top: 15px;
+    }
+`
+
+const HeroContent = styled(Flex)`
+    height: unset;
+
+    ${Header} {
+        display: flex;
+        margin-top: 1rem;
+    }
+
+    @media ${device.mobileL} {
+        ${Header} {
+            margin: 10px 0 0;
+        }
```

Comment:
Whenever possible, please try to avoid styling elements in this manner `${ElementName}` as it will override specificity over responsive style props.